pylab/settings/locale/lt/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,24 @@
+# This file is distributed under the same license as the pylab package.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-07-25 05:01-0500\n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: Lithuanian\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && (n"
+"%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#: base.py:24
+msgid "Lithuanian"
+msgstr "Lietuvių"
+
+#: base.py:25
+msgid "English"
+msgstr "Anglų"
```

Comment:
Interesting decision to create locale files for `pylab.settings`. I guess this is not going to work, since `pylab.settings` is not in `INSTALLED_APPS`.

I think is will be much easier if you create locale for whole `pylab` package, for all apps all together. Later you can specify this locale directory to be discovered by Django using [LOCALE_PATHS](https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-LOCALE_PATHS) setting.